.github/workflows/tests.yml

```diff
@@ -109,4 +110,5 @@ jobs:
           SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
           SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
           BROWSERS: preset:sauce
+          CONCURRENT_BROWSERS: 2
```

Comment:
What's the reason for the different values? Can you add a comment for future readers?

Reply:
Added comment.